### ISSUE TITLE
Aliases for Multisite

### ISSUE THREAD

**Issue by rroblik** [hechoendrupal/DrupalConsole#1851]
Hi,

After spend a lot of time to understand how to use Drupal Console on multi-site installation ...
I ask if we can configure aliases somewhere (like in `~/.composer/xxx.yml` ?) to be able to do like I did with Drush in the past : 

```
drupal @mysite.dev module:generate
```

I can't see any doc about that... can you help me ?

Regards

**Comment by jmolivas** [comment]
@rroblik:  We have sites aliases, but you need to use `--target` option, something like this:
```
$ drupal --target=sample.dev cr all
```
For docs on this topic, you can visit `How to use Drupal Console in a remote site installation`
https://hechoendrupal.gitbooks.io/drupal-console/content/en/using/how-to-use-drupal-console-in-a-remote-installation.html

We do not have any docs yet aboout, `How to configure a site alias` but so far this is what is required to do.  
Execute `drupal init`, this will copy configuration files to your home directory  
![drupal-init](https://cloud.githubusercontent.com/assets/366275/12759435/f35ad6bc-c995-11e5-8a05-6cfd1c66caa2.png)

As you can see a `sites/sample.yml` was copied you can copy/paste to customize it based on your needs.

**Comment by rroblik** [comment]
thanks @jmolivas I already saw that
I think add an *alias* item inside `mysite.yml` for each environment should be very good :) , something like that : 
```
#mysite.yml

local:
  root: /var/www/drupal8.dev
  host: local
  alias: local
dev:
  root: /var/www/html/drupal
  host: 140.211.10.62
  user: drupal
  alias: dev
prod:
  root: /var/www/html/docroot
  host: live.drupal.org
  user: drupal
  console: /var/www/html/docroot/console.phar
  alias: prod
```
Then `drupal mysite@dev cr` or `drupal @dev cr` (as Drush !) 
(`--target` syntax is... too long !)

Regard

**Comment by jmolivas** [comment]
Like the idea of the autolias as we used on `--target=sample.dev`. 

I need to confirm if doing something like `@sample.dev` or `sample@dev` is possible with `symfony/console` component. ( I kinda like the first one better `@sample.dev`)

**Comment by rroblik** [comment]
I agree with you :)

+1 for `@sample.dev` 

(maybe add a "feature" label to this topic)

**Comment by jmolivas** [comment]
@rroblik: I write some code and this is how it looks like  
![drupal-console-alias-format](https://cloud.githubusercontent.com/assets/366275/12762291/58d8681c-c9a3-11e5-8764-c3672f5bd26b.png)

**Comment by rroblik** [comment]
Very nice ! Available in dev??

**Comment by jmolivas** [comment]
@rroblik:  PR comming later on today.

**Comment by jmolivas** [comment]
@rroblik: Merged and included on new release v0.10.8.
